sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceApiTests.java

```diff
@@ -848,6 +875,11 @@ public void getAccountInfoMin() {
         assertResponseStatusCode(primaryBlobServiceClient.getAccountInfoWithResponse(null, null), 200);
     }
 
+    @Test
+    public void getAccountInfoSimple() {
+        assertDoesNotThrow(() -> primaryBlobServiceClient.getAccountInfo());
```

Comment:
Let's change this one to an `assertNotNull` assertion to make sure we get an object returned